src/backend/dx12/src/device.rs

```diff
@@ -726,7 +726,7 @@ impl d::Device<B> for Device {
         color_attachments: &[&n::RenderTargetView],
         depth_stencil_attachments: &[&n::DepthStencilView],
         _extent: d::Extent,
-    ) -> Result<n::FrameBuffer, d::FramebufferError> {
+    ) -> Result<n::FrameBuffer, d::FrameBufferError> {
```

Comment:
Let's keep it the old way. I believe saying `Framebuffer` and `Renderpass` consistently across core/backends would be nicer

Reply:
They are named `FrameBuffer` and `RenderPass` in the backend trait though.

Reply:
Let's rename the backend trait types then. I believe we did discuss this naming issue recently and agreed upon the Vulkan-like naming, right @msiglreith ?

Reply:
Note: we'll rename it outside of this PR. It's already big enough